pimd/pim_bsm.c

```diff
@@ -637,7 +645,7 @@ void pim_bsm_clear(struct pim_instance *pim)
 		rp_all = pim_rp_find_match_group(pim, &g_all);
 
 		if (rp_all == rp_info) {
-			rp_all->rp.rpf_addr.family = AF_INET;
+			rp_all->rp.rpf_addr.family = PIM_AF;
 			rp_all->rp.rpf_addr.u.prefix4.s_addr = INADDR_NONE;
```

Comment:
Only setting IPv4 here

Use `pim_addr_to_prefix(&rp_all->rp.rpf_addr, PIMADDR_NONE);`

Reply:
This is taken care in the PR where we will initialise to INADDR_ANY , that time I will use
pim_addr_to_prefix(&rp_all->rp.rpf_addr, PIMADDR_ANY);